packages/nimiq-validator-trustscore/build.config.ts

```diff
@@ -2,11 +2,11 @@ import { defineBuildConfig } from 'unbuild'
 
 export default defineBuildConfig({
   entries: [
-    './src/index',
+    'src/fetcher',
+    'src/score',
+    'src/types',
+    'src/range',
   ],
-  declaration: true,
+  declaration: 'node16',
```

Comment:
Adding a comment on why this is needed would be nice

Reply:
https://github.com/unjs/unbuild#configuration

```ts
    /**
     * * `compatible` means "src/index.ts" will generate "dist/index.d.mts", "dist/index.d.cts" and "dist/index.d.ts".
     * * `node16` means "src/index.ts" will generate "dist/index.d.mts" and "dist/index.d.cts".
     * * `true` is equivalent to `compatible`.
     * * `false` will disable declaration generation.
     * * `undefined` will auto detect based on "package.json". If "package.json" has "types" field, it will be `"compatible"`, otherwise `false`.
```